tests/meta/report/test_meta_report.py

```diff
@@ -64,12 +67,14 @@ def test_incorporate_delivery_date_from_lims(lims_samples, report_api):
 
     # WHEN fetch_and_add_delivery_date_from_lims
     samples = lims_samples
-    report_api._incorporate_delivery_date_from_lims(samples)
+    report_api._incorporate_dates_from_lims(samples)
 
     # THEN
     # each sample has a property processing_time with a value of the days between received_at and
```

Comment:
I don't understand how the comment is related to the test?

Reply:
fixed